src/pages/Login.tsx

```diff
@@ -0,0 +1,19 @@
+import { useContext, useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { FormLogin } from '../components/LoginForm'
+import { AuthContext } from '../context/AuthContext'
+
+export const LoginPage = () => {
+  const { isAuth } = useContext(AuthContext)
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    isAuth ? navigate('/') : ''
```

Comment:
prefer `if (isAuth) navigate('/')`, it's easier to read

Reply:
Ur right. Going to change it